src/main/scala/mesosphere/marathon/core/instance/GoalAdjustmentReason.scala

```diff
@@ -0,0 +1,30 @@
+package mesosphere.marathon
+package core.instance
+
+/**
+  * Enumeration for the reason why an instance's goal has been adjusted.
+  */
+sealed trait GoalAdjustmentReason extends Product with Serializable
+
+object GoalAdjustmentReason {
```

Comment:
Nit: I think `GoalChangeReason` goes more in line with `GoalChangedEvent`.

Reply:
Why is this resolved? Do you reject my proposal which is fine or am I missing the change?